src/App.jsx

```diff
@@ -14,70 +15,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {
+          selectedGood
+            ? `${selectedGood} is selected`
+            : 'No goods selected'
+        }
```

Comment:
```suggestion
        {selectedGood
          ? `${selectedGood} is selected`
          : 'No goods selected'
        }
```
Pay attention to the formatting